src/DocumentPicker.Plugin/Shared/DocumentConversionExtension.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using DocumentConverter.Plugin.Shared.StreamProvider;
+using PdfToSvg;
+using Svg;
+
+namespace DocumentConverter.Plugin.Shared
+{
+    public static class DocumentConversionExtension
```

Comment:
Also, I see there is a `DocumentConverterService` - so what do we need this extensions class for?